x-pack/plugins/apm/public/components/app/Settings/AgentConfigurations/index.tsx

```diff
@@ -42,15 +43,21 @@ export function AgentConfigurations() {
       <EuiTitle size="l">
         <h1>
           {i18n.translate('xpack.apm.agentConfig.titleText', {
-            defaultMessage: 'Agent remote configuration',
+            defaultMessage: 'Agent central configuration',
           })}
         </h1>
       </EuiTitle>
+      <EuiSpacer size="s" />
+      <EuiText color="subdued">
+        {i18n.translate('xpack.apm.settings.agentConfig.descriptionText', {
+          defaultMessage: `This allows you to fine-tune your agent configuration from within the APM app. Changes are automatically propagated to your APM agents, so there’s no need to redeploy.`,
```

Comment:
```suggestion
          defaultMessage: `Fine-tune your agent configuration from within the APM app. Changes are automatically propagated to your APM agents, so there’s no need to redeploy.`,
```

Reply:
I think you can simplify this by removing the first few words.

Reply:
Ahh, I see this was copied over from https://github.com/elastic/kibana/blame/master/x-pack/plugins/apm/public/components/app/Settings/AgentConfigurations/AgentConfigurationCreateEdit/index.tsx#L126-L128... I'd recommend the same change there :)

Reply:
Updated the agent config description here https://github.com/elastic/kibana/pull/94398/commits/b82909535cf8067ff4c6fc73c88514a5ac302efb